module/transformer/src/Domain/Model/Record.php

```diff
@@ -113,43 +113,45 @@ public function getTranslation(string $name): TranslatableString
      *
      * @return bool
      */
-    public function hasValue(string $name): bool
+    public function hasAttribute(string $name): bool
     {
-        if (array_key_exists($name, $this->values)) {
+        if (array_key_exists($name, $this->value)) {
             return true;
         }
 
         return false;
     }
 
     /**
-     * @param string $name
+     * @param string $code
      *
-     * @return ValueInterface|null
+     * @return string|null
      */
-    public function getValue(string $name): ?ValueInterface
+    public function getAttribute(string $code): ?string
```

Comment:
Why `getAttribute` is searched by `code` and `hasAttribute` is searched by `name`?